### PR TITLE
Update Home Page and Add New Section

### PR DESCRIPTION
**Pull request by isharailanga**
This PR is to do the following changes:

- Add a new section named “Security Vulnerabilities” and have “Report Security Issues” as the section’s landing page.
- Move the “Reward and Acknowledgement Program” section under Security Processes and Programs to the newly added “Security Vulnerabilities”’ section.
- Link the "Report Issues" button to the “Report Security Issues” page.
- Add a new tile for Rewards and Acknowledgements in the Home Page.
- Do we have a Youtube intro video for WSO2 Security? Anything that is WIP? 
add it under the vulnerability mgt process?
- Remove displaying the document version, published date, and author (as this is not practiced in Git docs, also adding a date gives the message that the content is outdated) - These details were not removed in the “Security Advisory” docs. 

screencapture-127-0-0-1-8000-2022-11-14-18_48_40

### CODE REVIEW

**Comment by isharailanga** [comment]
- Remove the "Version History Sections" from Secure Coding Guidelines
- Rename Security Vulnerabilities to "Security Reporting"
- Fix the broken link in OH GPG Key Button
- Remove version history from all docs except for "Security Announcements"
- Maintain only the version details in the "Security Guidelines" doc

File: en/mkdocs.yml [hunk 1]
```diff
@@ -45,13 +45,10 @@ theme:
...
-    - '': 'security-processes-programs/index.md'
-    - 'Secure Software Development Process': 'security-processes-programs/secure-software-development-process.md'
-    - 'Vulnerability Management Process': 'security-processes-programs/vulnerability-management-process.md'
-    - 'Reward and Acknowledgement Program':
-      - 'Reward and Acknowledgement Program': 'security-processes-programs/reward-and-acknowledgement-program/index.md'
-      - 'Security Hall of Fame': 'security-processes-programs/reward-and-acknowledgement-program/hall-of-fame.md'
+  - 'Security Processes':
+    - '': 'security-processes/index.md'
+    - 'Secure Software Development Process': 'security-processes/secure-software-development-process.md'
+    - 'Vulnerability Management Process': 'security-processes/vulnerability-management-process.md'
```
Comment (nilminiwso2) [inline en/mkdocs.yml:51]: Vulnerability reporting guidelines should also be moved under `Security Reporting`

File: en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 
-WSO2 is pleased to recognize the security researchers who have helped making WSO2 products and services safer by finding and responsibly reporting security vulnerabilities. Each name listed here, represents an individual or a company who has reported one or more security vulnerabilities in our products or services and worked with us to rectify the issue.
+WSO2 is pleased to recognize the security researchers who have helped in making WSO2 products and services safer by 
+finding and responsibly reporting security vulnerabilities. Each name listed here represents an individual or a company 
+that has reported one or more security vulnerabilities in our products or services and worked with us to rectify the issue.
 
-However, please note that WSO2 maintained websites (including wso2.com) are currently not considered for proceeding with acknowledgement. Refer our [Security Reward and Acknowledgement Program]({{#base_path#}}/security-processes-programs/reward-and-acknowledgement-program/) for knowing more on our security researcher community relationship.
+However, please note that WSO2 maintained websites (including wso2.com) are currently not considered for proceeding
```
Comment (nilminiwso2) [inline en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md:12]:
```suggestion
However, please note that WSO2-maintained websites (including wso2.com) are currently not considered for proceeding 
```

File: en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+WSO2 is pleased to recognize the security researchers who have helped in making WSO2 products and services safer by 
+finding and responsibly reporting security vulnerabilities. Each name listed here represents an individual or a company 
+that has reported one or more security vulnerabilities in our products or services and worked with us to rectify the issue.
 
-However, please note that WSO2 maintained websites (including wso2.com) are currently not considered for proceeding with acknowledgement. Refer our [Security Reward and Acknowledgement Program]({{#base_path#}}/security-processes-programs/reward-and-acknowledgement-program/) for knowing more on our security researcher community relationship.
+However, please note that WSO2 maintained websites (including wso2.com) are currently not considered for proceeding 
+with acknowledgement. Refer to our [Security Reward and Acknowledgement Program](index.md) to learn more about our
```
Comment (nilminiwso2) [inline en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md:13]: Is this link correct?

File: en/docs/security-reporting/reward-and-acknowledgement-program/index.md [hunk 4]
```diff
@@ -77,13 +74,13 @@ We review reported security issues case-by-case. Following are common issues tha
 To show our appreciation, we provide a reward and an acknowledgement to eligible reporters after the reported issues are fixed and announced to the WSO2 customers and the community users.
 
 !!! tip
-    Please refer to our [Vulnerability Management Process]({{#base_path#}}/security-processes-programs/vulnerability-management-process/) for more details about how we disclose security vulnerabilities.
+    Please refer to our [Vulnerability Management Process](../../security-processes/vulnerability-management-process.md) for more details about how we disclose security vulnerabilities.
```
Comment (nilminiwso2) [inline en/docs/security-reporting/reward-and-acknowledgement-program/index.md:77]:
```suggestion
    See our [Vulnerability Management Process](../../security-processes/vulnerability-management-process.md) for more details about how we disclose security vulnerabilities.
```

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md (hunk 3 above)
Comment (isharailanga) [reply en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md:13]: yes. In this repo, they have added a landing page for each subtopic by using an `index.md`. I followed the same approach.

File: en/docs/security-reporting/reward-and-acknowledgement-program/index.md (hunk 4 above)
Comment (isharailanga) [reply en/docs/security-reporting/reward-and-acknowledgement-program/index.md]: Fixed in 4cf2f78e9fde97193b615b7bda53de23d579955c

File: en/docs/security-reporting/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+title: Security Reporting
+category: security-reporting
+---
+
+# Security Reporting
+
+This section explains how to [report security issues](report-security-issues.md) to WSO2 products and 
+[get rewarded](reward-and-acknowledgement-program/index.md) for your contributions. 
+
+We have been recognizing the efforts of the security research community for helping us make WSO2 products and
```
[the remark this answers is not in the document]
Comment (isharailanga) [reply en/docs/security-reporting/index.md]: Fixed in 4cf2f78e9fde97193b615b7bda53de23d579955c

File: en/docs/security-reporting/index.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+title: Security Reporting
+category: security-reporting
+---
+
+# Security Reporting
+
+This section explains how to [report security issues](report-security-issues.md) to WSO2 products and
```
[the remark this answers is not in the document]
Comment (isharailanga) [reply en/docs/security-reporting/index.md]: Fixed in 4cf2f78e9fde97193b615b7bda53de23d579955c

File: en/mkdocs.yml (hunk 1 above)
Comment (isharailanga) [reply en/mkdocs.yml:51]: Fixed in 4cf2f78e9fde97193b615b7bda53de23d579955c

**Review by nilminiwso2 — APPROVED** [review]

**Review by nilminiwso2 — APPROVED** [review]

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**